vite.config.ts

```diff
@@ -12,6 +12,7 @@ export default () => {
       rollupOptions: {
         maxParallelFileOps: 10,
       },
+      sourcemap: false,
```

Comment:
What does this do?

Reply:
Essentially nothing, the same as not having it there. But I am planning on having that boolean get flipped on and off for some builds to make debugging easier.